### PR TITLE
Update machine config

### PR DESCRIPTION
**Pull request by omrirh**
Updating MachineConfig existing resource with proper attributes as described in the official Openshift documentation.

### CODE REVIEW

**Comment by myakove** [comment]
@omrirh Please add a proper message and description for this PR

File: ocp_resources/machine_config.py [hunk 1]
```diff
@@ -1,5 +1,93 @@
...
+
     api_group = Resource.ApiGroup.MACHINECONFIGURATION_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        os_image_url: Optional[str] = "",
+        config: Optional[Dict[str, Any]] = None,
+        kernel_arguments: Optional[List[str]] = None,
+        extensions: Optional[List[str]] = None,
+        fips: Optional[bool] = False,
```
Comment (myakove) [inline ocp_resources/machine_config.py:18]: We do not set defaults, please set to None and use it only if the userr pass it.

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/machine_config.py (hunk 1 above)
Comment (omrirh) [reply ocp_resources/machine_config.py]: done

**Comment by msi-b1** [comment]
oharan2 can not be added as reviewer. 422 {"message": "Reviews may only be requested from collaborators. One or more of the users or teams you specified is not a collaborator of the RedHatQE/openshift-python-wrapper repository.", "documentation_url": "https://docs.github.com/rest/pulls/review-requests#request-reviewers-for-a-pull-request", "status": "422"}

**Comment by myakove** [comment]
This `Required for https://issues.redhat.com/browse/MSINTEROP-873 day 2 operations` is not a proper commit message.

Please describe what the PR do, not why it is required

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/machine_config.py [hunk 2]
```diff
@@ -1,5 +1,93 @@
+from typing import Any, Dict, List, Optional
 from ocp_resources.resource import Resource
 
 
 class MachineConfig(Resource):
+    """
+    https://docs.openshift.com/container-platform/4.15/rest_api/machine_apis/machineconfig-machineconfiguration-openshift-io-v1.html
```
[the remark this answers is not in the document]
Comment (omrirh) [reply ocp_resources/machine_config.py]: done

**Review by rnetser — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**